File: github/Programatres.java

package programatres;
 import java.util.Scanner;
public class Programatres {

  
    public static void main(String[] args) {
double a, l;
        Scanner scanner = new Scanner(System.in);
        System.out.print("Introduce el valor del lado del hexágono: ");
        l = scanner.nextDouble();
        a = (3 * Math.sqrt(3) / 2) * (l * l);
        System.out.print("El área del hexágono es: " + a);
    }
    
}

File: github/Programatres.cs

using System;
namespace programaocho
{
    class programaocho
    {
        static void Main()
        {
            double a,l;
            Console.WriteLine("intorduce el valor del lado del hexagono");
            l=Convert.ToDouble (Console.ReadLine());
            a = (3 * Math.Sqrt(3) / 2) * (l * l);
            Console.WriteLine("el area del heaxogono es" + a);
        }
    }
}
